Gateway/Request/APMBuilder.php

```diff
@@ -375,16 +375,17 @@ private function getOrderItems($order)
         $itemArray = [];
         $items = $order->getItems();
         $currency = $order->getCurrencyCode();
-        foreach ($items as $itemObject) {
-            $item = $itemObject->toArray();
+
+        foreach ($items as $item) {
+            $price = $item->getPrice();
+            if ((float) $price === 0.0) {
+                continue;
+            }
             $itemArray[] = [
-                'sku' => $item['sku'],
-                'name' => $item['name'],
-                'amount' => $this->money->setAmountAndCurrency(
-                    $item['base_original_price'],
-                    $currency
-                )->toSubunit(),
-                'quantity' => $item['qty_ordered'],
+                'sku' => $item->getSku(),
+                'name' => $item->getName(),
+                'amount' => $this->money->setAmountAndCurrency($price, $currency)->toSubunit(),
```

Comment:
Did a linter trigger the formatting change?

Reply:
No. Currently we do not have linter in Magento.

Reply:
But we do have Magento 2 coding standard checker in our pipeline.

Reply:
Oh, it would be nice to have a linter so our formatting changes are guaranteed to only go one way.

Reply:
The file is formatted with php-cs-fixer locally. 
Will add linter with separate ticket in order to share the same rule with team.

Reply:
Great, thanks.